### ISSUE TITLE
NoMethodError when editing a NewspaperIssue

### ISSUE THREAD

**Issue by ebenenglish** [marriott-library/newspaper_works#133]
**To reproduce:**
1. Create a `NewspaperIssue` object with a PDF file upload
2. Once the item has been ingested, go to the show page for the issue
3. Click 'Edit'
4. From the edit form, click 'Save'

**Error:**
```
NoMethodError in Hyrax::NewspaperIssuesController#update
undefined method `empty?' for nil:NilClass
```
Looks like `@pdf_paths` is not getting set in `newspaper_works_upload_actor.rb:66`.

**Full trace:**
```
/home/vagrant/newspaper_works/app/actors/newspaper_works/actors/newspaper_works_upload_actor.rb:66:in `handle_issue_upload'
/home/vagrant/newspaper_works/app/actors/newspaper_works/actors/newspaper_works_upload_actor.rb:38:in `update'
hyrax (2.5.0) app/actors/hyrax/actors/create_with_remote_files_actor.rb:23:in `update'
hyrax (2.5.0) app/actors/hyrax/actors/optimistic_lock_validator.rb:13:in `update'
hyrax (2.5.0) app/actors/hyrax/actors/transactional_request.rb:19:in `update'
hyrax (2.5.0) app/controllers/concerns/hyrax/works_controller_behavior.rb:98:in `update'
actionpack (5.1.6.2) lib/action_controller/metal/basic_implicit_render.rb:4:in `send_action'
actionpack (5.1.6.2) lib/abstract_controller/base.rb:186:in `process_action'
actionpack (5.1.6.2) lib/action_controller/metal/rendering.rb:30:in `process_action'
actionpack (5.1.6.2) lib/abstract_controller/callbacks.rb:20:in `block in process_action'
activesupport (5.1.6.2) lib/active_support/callbacks.rb:131:in `run_callbacks'
actionpack (5.1.6.2) lib/abstract_controller/callbacks.rb:19:in `process_action'
actionpack (5.1.6.2) lib/action_controller/metal/rescue.rb:20:in `process_action'
actionpack (5.1.6.2) lib/action_controller/metal/instrumentation.rb:32:in `block in process_action'
activesupport (5.1.6.2) lib/active_support/notifications.rb:166:in `block in instrument'
activesupport (5.1.6.2) lib/active_support/notifications/instrumenter.rb:21:in `instrument'
activesupport (5.1.6.2) lib/active_support/notifications.rb:166:in `instrument'
actionpack (5.1.6.2) lib/action_controller/metal/instrumentation.rb:30:in `process_action'
actionpack (5.1.6.2) lib/action_controller/metal/params_wrapper.rb:252:in `process_action'
activerecord (5.1.6.2) lib/active_record/railties/controller_runtime.rb:22:in `process_action'
actionpack (5.1.6.2) lib/abstract_controller/base.rb:124:in `process'
actionview (5.1.6.2) lib/action_view/rendering.rb:30:in `process'
actionpack (5.1.6.2) lib/action_controller/metal.rb:189:in `dispatch'
actionpack (5.1.6.2) lib/action_controller/metal.rb:253:in `dispatch'
actionpack (5.1.6.2) lib/action_dispatch/routing/route_set.rb:49:in `dispatch'
actionpack (5.1.6.2) lib/action_dispatch/routing/route_set.rb:31:in `serve'
actionpack (5.1.6.2) lib/action_dispatch/journey/router.rb:50:in `block in serve'
actionpack (5.1.6.2) lib/action_dispatch/journey/router.rb:33:in `each'
actionpack (5.1.6.2) lib/action_dispatch/journey/router.rb:33:in `serve'
actionpack (5.1.6.2) lib/action_dispatch/routing/route_set.rb:844:in `call'
warden (1.2.8) lib/warden/manager.rb:36:in `block in call'
warden (1.2.8) lib/warden/manager.rb:34:in `catch'
warden (1.2.8) lib/warden/manager.rb:34:in `call'
rack (2.0.6) lib/rack/etag.rb:25:in `call'
rack (2.0.6) lib/rack/conditional_get.rb:38:in `call'
rack (2.0.6) lib/rack/head.rb:12:in `call'
rack (2.0.6) lib/rack/session/abstract/id.rb:232:in `context'
rack (2.0.6) lib/rack/session/abstract/id.rb:226:in `call'
actionpack (5.1.6.2) lib/action_dispatch/middleware/cookies.rb:613:in `call'
active-fedora (11.5.4) lib/active_fedora/ldp_cache.rb:26:in `call'
flipflop (2.4.0) lib/flipflop/feature_cache.rb:12:in `call'
activerecord (5.1.6.2) lib/active_record/migration.rb:556:in `call'
actionpack (5.1.6.2) lib/action_dispatch/middleware/callbacks.rb:26:in `block in call'
activesupport (5.1.6.2) lib/active_support/callbacks.rb:97:in `run_callbacks'
actionpack (5.1.6.2) lib/action_dispatch/middleware/callbacks.rb:24:in `call'
actionpack (5.1.6.2) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.1.6.2) lib/action_dispatch/middleware/debug_exceptions.rb:59:in `call'
web-console (3.7.0) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.7.0) lib/web_console/middleware.rb:22:in `block in call'
web-console (3.7.0) lib/web_console/middleware.rb:20:in `catch'
web-console (3.7.0) lib/web_console/middleware.rb:20:in `call'
actionpack (5.1.6.2) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.1.6.2) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.1.6.2) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.1.6.2) lib/active_support/tagged_logging.rb:69:in `block in tagged'
activesupport (5.1.6.2) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.1.6.2) lib/active_support/tagged_logging.rb:69:in `tagged'
railties (5.1.6.2) lib/rails/rack/logger.rb:24:in `call'
sprockets-rails (3.2.1) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.1.6.2) lib/action_dispatch/middleware/remote_ip.rb:79:in `call'
actionpack (5.1.6.2) lib/action_dispatch/middleware/request_id.rb:25:in `call'
rack (2.0.6) lib/rack/method_override.rb:22:in `call'
rack (2.0.6) lib/rack/runtime.rb:22:in `call'
activesupport (5.1.6.2) lib/active_support/cache/strategy/local_cache_middleware.rb:27:in `call'
actionpack (5.1.6.2) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.1.6.2) lib/action_dispatch/middleware/static.rb:125:in `call'
actionpack (5.1.6.2) lib/action_dispatch/middleware/static.rb:125:in `call'
rack (2.0.6) lib/rack/sendfile.rb:111:in `call'
railties (5.1.6.2) lib/rails/engine.rb:522:in `call'
puma (3.12.1) lib/puma/configuration.rb:227:in `call'
puma (3.12.1) lib/puma/server.rb:660:in `handle_request'
puma (3.12.1) lib/puma/server.rb:474:in `process_client'
puma (3.12.1) lib/puma/server.rb:334:in `block in run'
puma (3.12.1) lib/puma/thread_pool.rb:135:in `block in spawn_thread'
```

**Comment by ebenenglish** [comment]
This also apparently applies to NDNP-batch-ingested Issue objects. Same error when attempting to edit a saved Issue.

**Comment by ebenenglish** [comment]
Fixed via #177
